deployment/playbooks/roles/aws/templates/env.j2

```diff
@@ -0,0 +1,17 @@
+DJANGO_DEBUG="False"
+HOUNDIGRADE_AWS_AVAILABILITY_ZONE="{{ec2_asg_az}}"
+HOUNDIGRADE_AWS_AUTOSCALING_GROUP_NAME="{{ec2_autoscaling_group.auto_scaling_group_name}}"
+HOUNDIGRADE_ECS_CLUSTER_NAME="{{ecs_cluster.cluster.clusterName}}"
+
+AWS_ACCESS_KEY_ID="{{iam_user.user_meta.access_keys[0].access_key_id}}"
+AWS_SECRET_ACCESS_KEY="{{iam_user.user_meta.access_keys[0].secret_access_key}}"
+
+AWS_SQS_ACCESS_KEY_ID="{{iam_user.user_meta.access_keys[0].access_key_id}}"
+AWS_SQS_SECRET_ACCESS_KEY="{{iam_user.user_meta.access_keys[0].secret_access_key}}"
+
+AWS_NAME_PREFIX="{{env}}-"
+
+AWS_CLOUDTRAIL_EVENT_URL="{{ sqs_queue.queue_arn | regex_replace('^arn.+sqs.+us-east-1:(?P<account_id>\d+):(?P<sqs_queue_name>\S+)$', 'https://sqs.us-east-1.amazonaws.com/\\g<account_id>/\\g<sqs_queue_name>')}}"
```

Comment:
This looks like it works fine, but I'm trying to think if there's a more elegant way to set this value.

Do you think there's something we can do with [amazon.aws.aws_caller_info](https://docs.ansible.com/ansible/latest/collections/amazon/aws/aws_caller_info_module.html#ansible-collections-amazon-aws-aws-caller-info-module) to get the AWS account ID without having to parse apart an ARN like this?

Reply:
Also I'm wondering if we should make the region configurable, though I know we also have `us-east-1` and `us-east-1a` baked in several other places.

Reply:
There is no elegant way to set this value because the aws module fails to return the url like it is supposed to half the time. I don't really see the value in having an extra call to fetch information that we can otherwise put together with what was already returned.

Additionally, this file is unused until we have shared storage between pods.